masterlist.yaml

```
@@ -450,24 +450,24 @@ common:
   - &patch3rdParty_RSCPC
     <<: *patch3rdParty
     subs:
-      - 'RS Children Overhaul'
+      - '[RS Children Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/2650/)'
       - '[RS Children Patch Compendium](https://www.nexusmods.com/skyrimspecialedition/mods/13409/)'
   - &patch3rdPartySBCK
     <<: *patch3rdParty
     subs:
       - '[Scoped Bows](https://www.nexusmods.com/skyrimspecialedition/mods/912/)'
-      - '[Scoped Bows Compatibility Kit - SBCK](https://www.nexusmods.com/skyrimspecialedition/mods/19495)'
+      - '[Scoped Bows Compatibility Kit - SBCK](https://www.nexusmods.com/skyrimspecialedition/mods/19495/)'
   - &patch3rdPartySRPC
     <<: *patch3rdParty
     subs:
-      - '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
+      - '[Spell Research](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
```

The patches on that page are specifically for version 2.2 & up of that mod, so we should specify that in the sub.
```suggestion
      - '[Spell Research v2.2+](https://www.nexusmods.com/skyrimspecialedition/mods/20983/)'
```